zabbix-ldap-sync

```diff
@@ -106,21 +106,24 @@ class LDAPConn(object):
                 result_dn = members[0]
                 result_attrs = members[1]
 
-            if "member" in result_attrs:
-                for member in result_attrs["member"]:
+            	if "member" in result_attrs:
+                	for member in result_attrs["member"]:
+		    		member = member.replace("(","\\28")
+		    		member = member.replace(")","\\29")
 
-                    attrlist = ['sAMAccountName']
-                    filter   = '(&(objectClass=person)(distinguishedName=%s))' % member
+                    		attrlist = ['userPrincipalName']
```

Comment:
This indentation doesn't seem right. Please try to follow PEP-8 guidelines about indentation.